GoodDay/Program.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+
+namespace GoodDay
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            Greeting greeting = new Greeting();
+            if (greeting.Hours(DateTime.Now.TimeOfDay) < 12 && greeting.Hours(DateTime.Now.TimeOfDay) >= 9)
```

Comment:
correct but better to put it like this:
`if (greeting.Hours(DateTime.Now.TimeOfDay) >= 9 &&  greeting.Hours(DateTime.Now.TimeOfDay) < 12)`